src/test/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParserTest.java

```diff
@@ -160,6 +159,19 @@ public void testBetaExperimentalMutex() {
         final CommandLineArgumentParser clp = new CommandLineArgumentParser(eo);
     }
 
+    class ArrayArgument {
```

Comment:
May want to make this test class `private`.

Also all the other test classes in this test should be made `private`.

Reply:
Done.